Sources/NIO/EventLoopFuture.swift

```diff
@@ -894,10 +896,13 @@ extension EventLoopFuture {
     ///     - eventLoop: The `EventLoop` on which the new `EventLoopFuture` callbacks will fire.
     ///     - nextPartialResult: The bifunction used to produce partial results.
     /// - returns: A new `EventLoopFuture` with the reduced value.
-    public static func reduce<U>(_ initialResult: T, _ futures: [EventLoopFuture<U>], eventLoop: EventLoop, _ nextPartialResult: @escaping (T, U) -> T) -> EventLoopFuture<T> {
+    public static func reduce<U>(_ initialResult: Value,
```

Comment:
Same question here about `U`.

Reply:
thanks, renamed to `NewValue`, `OtherValue`, or `InputValue` depending on the context.